docs/Windows.md

```diff
@@ -1,216 +1,29 @@
 # Getting Started with Swift on Windows
 
-## clang (cross-compiling)
+This document summarizes the state of Swift on Windows.
 
-### 1. Setup Visual Studio Environment Variables
-Building for Windows requires that the Visual Studio environment variables are
-setup similar to the values on Windows. The following assumes that
-`WINKIT_ROOT` points to the path where the Windows 10 SDK is available and that
-`VC_ROOT` points to the path where the Visual Studio VC headers and libraries
-are available. Currently, the runtime has been tested to build against the
-Windows 10 SDK at revision 10.10.586.
+## Native Windows
 
-```bash
-# Visual Studio 2015 does not have VCToolsInstallDir, use VCINSTALLDIR's value
-export UCRTVersion=10.0.10586.0
-export UniversalCRTSdkDir=".../Windows Kits/10"
-export VCToolsInstallDir=".../Microsoft Visual Studio/2017/Community"
-```
+Currently there are 3 supported ways to build Swift for Windows.
 
-### 2. Setup `visualc` and `ucrt` modules
-The `visualc.modulemap` located at
-`swift/stdlib/public/Platform/visualc.modulemap` needs to be copied into
-`${VCToolsInstallDir}/include`. The `ucrt.modulemap` located at
-`swift/stdlib/public/Platform/ucrt.modulemap` needs to be copied into
-`${UniversalCRTSdkDir}/Include/${UCRTVersion}/ucrt`.
+1. To cross-compile Swift for Windows from another host using `clang`, see
```

Comment:
The idea of cross-compiling isn't particularly clear if you're not familiar with it, so it might be good to be more explicit here. Something like "to cross-compile Swift for Windows targets from a different host operating system", and maybe add a comment like "Windows Subsystem for Linux provides a convenient way to do this".